### ISSUE TITLE
"Upload Text Files" has no "% for validation" / "% for testing"

### ISSUE THREAD

**Issue by cancan101** [NVIDIA/DIGITS#107]
Right now the "Upload Text Files" option when building a data set has no "% for validation" / "% for testing". ie I have to pre-split the data myself if I want to use that option.

**Comment by lukeyeager** [comment]
True. I'm assuming that if you're already building your own textfiles, then you can figure out how to do the split yourself. This would be a low-priority new feature.
